src/apps/chatter/components/chatbox/chatbox.js

```diff
@@ -17,17 +26,19 @@ export const ChatBox = ({ roomId }) => {
   }
 
   return (
-    <div className='chatbox-component'>
+    <div className='chatbox'>
 
       {messages.map((entry) => <Message content={entry.text} author={entry.author} timestamp={entry.timestamp} />)}
```

Comment:
Just noticed this, why not keep all the same keys `content => text` and then you can just go: 

```
      {messages.map(Message)}
```

Reply:
✔️